datafusion/sql/src/select.rs

```diff
@@ -59,6 +59,9 @@ impl<'a, S: ContextProvider> SqlToRel<'a, S> {
         if select.top.is_some() {
             return Err(DataFusionError::NotImplemented("TOP".to_string()));
         }
+        if !select.sort_by.is_empty() {
```

Comment:
👍 

It would be nice to add a .slt test for this case, but I also think it is ok not to